pkg/cmd/secret/set/set.go

```diff
@@ -209,12 +217,22 @@ func validSecretName(name string) error {
 
 func getBody(opts *SetOptions) ([]byte, error) {
 	if opts.Body == "" {
-		body, err := ioutil.ReadAll(opts.IO.In)
-		if err != nil {
-			return nil, fmt.Errorf("failed to read from STDIN: %w", err)
-		}
+		if opts.IO.CanPrompt() {
+			err := prompt.SurveyAskOne(&survey.Password{
+				Message: "Paste your secret",
+			}, &opts.Body)
+			if err != nil {
+				return nil, err
+			}
+			fmt.Fprintln(opts.IO.Out)
```

Comment:
Doesn't Survey take care of printing the newline after the prompt?

Reply:
usually yes, but i was seeing bizarre behavior with the password prompt. It printed a newline and then some tab characters and then output continued after the tabs. Manually printing a new line fixed it 🤷